INSTRUCTIONS.md

```diff
@@ -0,0 +1,15 @@
+In order to deploy the required cluster resources:
+./bootstrap.sh
+
+In order to check whether the todoapp server pod is running:
+kubectl get pods
+
+In order to check whether the ConfigMap and Secrets volumes are mounted correctly:
+
+kubectl exec -it <pod-name> -- /bin/sh
```

Comment:
Replace `<pod-name>` with the actual name of the pod running the todoapp server when executing this command.